sys/include/net/sock/dtls.h

```diff
@@ -547,25 +567,18 @@ int sock_dtls_create(sock_dtls_t *sock, sock_udp_t *udp_sock,
                      credman_tag_t tag, unsigned version, unsigned role);
 
 /**
- * @brief Creates a new DTLS session
- *
- * Initializes handshake process with a DTLS server  at @p ep.
+ * @brief Send a ClientHello to start the handshake
```

Comment:
```suggestion
 * @brief Send a ClientHello message to start the handshake
```

However, I would prefer to keep it abstract in the `@brief`:

```
Initialize session handshake.
```

It should also be explained in the detailed section of this function, that `sock_dtls_recv()` must be called to finish the handshake.